libs/coin-modules/coin-aptos/src/__tests__/network/client.test.ts

```diff
@@ -621,4 +628,450 @@ describe("Aptos API", () => {
+                    counter: "0",
+                    guid: {
+                      id: {
+                        addr: address,
+                        creation_num: "1",
+                      },
+                    },
+                  },
+                  rotation_capability_offer: {
+                    for: {
+                      vec: [],
+                    },
+                  },
+                  sequence_number: "121",
+                  signer_capability_offer: {
+                    for: {
+                      vec: [],
+                    },
+                  },
+                },
+              },
+              type: "write_resource",
+            },
+            {
+              state_key_hash: "0x6e4b",
+              handle: "0x1b85",
+              key: "0x0619",
+              value: "0xe86e0039581497010000000000000000",
+              type: "write_table_item",
+            },
+          ],
+          sender: address,
+          sequence_number: "120",
+          max_gas_amount: "12",
+          gas_unit_price: "100",
+          expiration_timestamp_secs: "1743176706",
+          payload: {
+            function: "0x1::aptos_account::transfer_coins",
+            type_arguments: ["0xd111::staked_coin::StakedAptos"],
+            arguments: ["0x4e5e", "2500000"],
+            type: "entry_function_payload",
+          },
+          events: [
+            {
+              guid: {
+                creation_number: "11",
+                account_address: address,
+              },
+              sequence_number: "11",
+              type: "0x1::coin::WithdrawEvent",
+              data: {
+                amount: "2500000",
+              },
+            },
+            {
+              guid: {
+                creation_number: "4",
+                account_address: "0x4e5e",
+              },
+              sequence_number: "7",
+              type: "0x1::coin::DepositEvent",
+              data: {
+                amount: "2500000",
+              },
+            },
+            {
+              guid: {
+                creation_number: "0",
+                account_address: "0x0",
+              },
+              sequence_number: "0",
+              type: "0x1::transaction_fee::FeeStatement",
+              data: {
+                execution_gas_units: "6",
+                io_gas_units: "6",
+                storage_fee_octas: "0",
+                storage_fee_refund_octas: "0",
+                total_charge_gas_units: "12",
+              },
+            },
+          ],
+          timestamp: "1743176594693251",
+          type: TransactionResponseType.User,
+          block: {
+            height: 311942427,
+            hash: "0x8655",
+          },
+        },
+      ];
+
+      const transactions: AptosTransaction[] = txs;
+
+      api.getAccountInfo = jest.fn().mockResolvedValue({ transactions });
+
+      const ops = await api.listOperations(address, pagination);
+
+      expect(ops[0]).toHaveLength(2);
```

Comment:
This expectation can be strengthen:
```suggestion
      expect(ops[0]).toEqual([
        expect.objectContaining(...),
        expect.objectContaining(...),
      ]);
```